docs/lib/datastore/fragments/js/data-access/observe-snippet.md

```diff
@@ -4,8 +4,18 @@ const subscription = DataStore.observe(Post).subscribe(msg => {
 });
 ```
 
+Observing changes of a single item by ID.
+
+```js
+const id = '69ddcb63-7e4a-4325-b84d-8592e6dac07b';
```

Comment:
nit: we've been using `"123"` as a representation of ids in the rest of the examples, maybe we should keep it consistent with those?

Reply:
Hey Daniel, thank you for pointing this out, I'll update in my next PR.